src/workflows/workflows.ts

```diff
@@ -65,6 +67,9 @@ export class Workflows {
   private readonly usersApi: UsersApi;
   private readonly withdrawalsApi: WithdrawalsApi;
 
+  private readonly isChainValid = (chainID: number) =>
```

Comment:
All right. Fixed that.

Reply:
@matthewmuscat  do you think it would be worth adding this eslint plugin to catch `this` usage in arrow functions?
https://github.com/getify/eslint-plugin-proper-arrows